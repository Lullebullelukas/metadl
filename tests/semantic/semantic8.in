F().
H(1) :- G().