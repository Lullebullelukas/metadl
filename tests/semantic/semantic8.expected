Nullary literals are not (yet) supported: F.
Nullary literals are not (yet) supported: G.